go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/btcsuite/btcd v0.0.0-20171128150713-2e60448ffcc6
 	github.com/ethereum/go-ethereum v1.9.9
 	github.com/golang/protobuf v1.4.2
-	github.com/onflow/cadence v0.9.1
-	github.com/onflow/flow/protobuf/go/flow v0.1.7
+	github.com/onflow/cadence v0.10.0
+	github.com/onflow/flow/protobuf/go/flow v0.1.8-0.20201002204652-a7175c92529c
```

Comment:
We should never use commits of branches, because they get deleted.

Commit ca151b05acef0c2d802f92bc5f2a15b663c17c67 in https://github.com/onflow/flow should get tagged and referenced here

Reply:
Tagged that repo and updated the dependency in 882d2e0